src/operator/tensor/elemwise_unary_op_trig.cc

```diff
@@ -92,7 +92,7 @@ The storage type of ``cos`` output is always dense
 MXNET_OPERATOR_REGISTER_BINARY_WITH_SPARSE_CPU(_backward_cos, unary_bwd<mshadow_op::cos_grad>)
 .set_attr<nnvm::FGradient>("FGradient",
     [](const nnvm::NodePtr& n, const std::vector<nnvm::NodeEntry>& ograds) {
-      // ograds[0]: head_grad_grads (dL/dx_grad)
+      // ograds[0]: head_grad_grads (dL/dy_grad)
```

Comment:
Why change this? I think the original dL/dx_grad is correct.

Reply:
Please have a look and let me know what you think.
https://github.com/apache/incubator-mxnet/pull/15331#discussion_r304541519